srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -119,6 +125,10 @@
 	    			"aTargets": [0]
 	    		}]
 	    	});
+        	// Use the CPT code for the proper radio button and set it to checked
+        	$("input:radio[value='${selectedProcedure}']").attr("checked",true);
```

Comment:
Also, you could just generate the hidden input in the JSP now and read its value in the JavaScript instead generating a dynamic selector like this.